### ISSUE TITLE
Disabling eslint for a line doesn't work with max-len

### ISSUE THREAD

**Issue by mattixittam** [vuejs/eslint-plugin-vue#419]
**Tell us about your environment**
When using //eslint-disable-line in the script section of a .vue-file, the max-len rule is still enforced in Atom.

When I don't use eslint-plugin-vue, it does work.

* **ESLint Version: 3.19.0**
* **eslint-plugin-vue Version: 4.3.0**
* **Node Version: 8.9.4**

**Please show your full configuration:**

```
module.exports = {
    root: true,
    parserOptions: {
        parser: 'babel-eslint',
        sourceType: 'module'
    },
    env: {
        browser: true,
    },
    extends: [
        'airbnb-base',
        'plugin:vue/recommended',
    ],
    // check if imports actually resolve
    'settings': {
        'import/resolver': {
            'webpack': {
                'config': 'config/webpack/webpack.base.conf.js'
            }
        }
    },
    // add your custom rules here
    'rules': {
        // don't require .vue extension when importing
        'import/extensions': ['error', 'always', {
            'js': 'never',
            'vue': 'never'
        }],
        // allow optionalDependencies
        'import/no-extraneous-dependencies': ['error', {
            'optionalDependencies': ['test/unit/index.js']
        }],
        // allow debugger during development
        'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
        'semi': ['error', 'never'],
        'indent': ['error', 4, {
          SwitchCase: 1,
          VariableDeclarator: 1,
          outerIIFEBody: 1,
          // MemberExpression: null,
          // CallExpression: {
          //   parameters: null,
          // },
          FunctionDeclaration: {
            parameters: 1,
            body: 1
          },
          FunctionExpression: {
            parameters: 1,
            body: 1
          }
        }],
        'no-mixed-operators': [
            'error',
            {
                "allowSamePrecedence": true
            }
        ],
        'max-len': ['error', 120],
        // eslint-plugin-vue
        "vue/html-indent": ["error", 4, {
            "attribute": 1,
            "alignAttributesVertically": false,
        }],
        "vue/html-self-closing": ["error", {
            "html": {
              "void": "always",
              "normal": "never",
              "component": "any"
            },
        }],
        "vue/max-attributes-per-line": [2, {
            "singleline": 2,
            "multiline": {
              "max": 1,
              "allowFirstLine": true
            }
        }],
        "vue/name-property-casing": ["error", "kebab-case"],
        "vue/order-in-components": ["error", {
          order: [
            "el",
            "name",
            "parent",
            "functional",
            ["delimiters", "comments"],
            "components",
            "directives",
            "filters",
            "extends",
            "mixins",
            "inheritAttrs",
            "model",
            ["props", "propsData"],
            "notReactive",
            "data",
            "validations",
            "computed",
            "watch",
            "LIFECYCLE_HOOKS",
            "methods",
            ["template", "render"],
            "renderError"
          ]
      }],
      "vue/html-closing-bracket-newline": ["error", {
        "singleline": "never",
        "multiline": "always"
      }],
      "vue/attributes-order": [2, {
          order: [
              "DEFINITION",
              "LIST_RENDERING",
              "CONDITIONALS",
              "RENDER_MODIFIERS",
              "GLOBAL",
              "UNIQUE",
              "CONTENT",
              "OTHER_ATTR",
              "BINDING",
              "EVENTS"
          ]
      }]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<script>
    methods: {
        testMethod() {
            const test = 'This is a long line, too long for max-len 120 including the comment, too short without.' //eslint-disable-line
        }
    }
</script>
```

**What did you expect to happen?**
The line should not be flagged as faulty.

**What actually happened? Please include the actual, raw output from ESLint.**
http://eslint.org/docs/rules/max-len  Line 3 exceeds the maximum line length of 120

**Comment by mattixittam** [comment]
Figured it out, the older version of eslint was the issue. I upgraded to 4.18.2 and it now works.
